flow/StreamCipher.cpp

```diff
@@ -56,8 +59,8 @@ StreamCipherKey const* StreamCipherKey::getGlobalCipherKey() {
 }
 
 void StreamCipherKey::cleanup() noexcept {
-	for (auto cipherKey : cipherKeys) {
-		cipherKey->reset();
+	for (auto itr : cipherKeys) {
```

Comment:
Nit: The name `itr` is a bit misleading here since `itr` is not an iterator. It's a `std::pair<UID, StreamCipherKey*>`. I'd also prefer the loop variable to be `const auto&` here.

Reply:
Technically the loop is iterating over a 'unordered_map'; so I think using "itr" or "mapItr" isn't too bad. However, I agree with using "const auto&" for sure. Thanks